index.html

```diff
@@ -0,0 +1,320 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
```

Comment:
We should also use the same href for the bootstrap.min.css everywhere right? Do we need this cdn or is it enough with the stackpath.bootstrapcdn?